lib/src/Base/Func/EvaluationImplementation.cxx

```diff
@@ -425,23 +426,11 @@ Graph EvaluationImplementation::draw(const UnsignedInteger firstInputMarginal,
   const String yName(getInputDescription()[secondInputMarginal]);
   String title(OSS() << getOutputDescription()[outputMarginal] << " as a function of (" << xName << "," << yName << ")");
   if (centralPoint.getDimension() > 2) title = String(OSS(false) << title << " around " << centralPoint);
-  Graph graph(title, xName, yName, true, "upper left", 1.0, scale);
-  graph.setLegendCorner({1.0, 1.0});
-  Contour isoValues(Contour(x, y, z, Point(0), Description(0), true, title));
-  isoValues.buildDefaultLevels();
-  isoValues.buildDefaultLabels();
-  const Point levels(isoValues.getLevels());
-  const Description labels(isoValues.getLabels());
-  for (UnsignedInteger i = 0; i < levels.getDimension(); ++i)
-  {
-    Contour current(isoValues);
-    current.setLevels(Point(1, levels[i]));
-    current.setLabels(Description(1, labels[i]));
-    current.setDrawLabels(false);
-    current.setLegend(labels[i]);
-    current.setColor(Contour::ConvertFromHSV((360.0 * i / levels.getDimension()), 1.0, 1.0));
-    graph.add(current);
-  }
+  Graph graph(title, xName, yName, true, "", 1.0, scale);
+  Contour isoValues(Contour(x, y, z, isFilled));
+  isoValues.setColorBarPosition("right");
```

Comment:
I suggest to compute default levels based on quantiles, since the levels are not provided as an input argument of this particular constructor. 
```suggestion
  isoValues.buildDefaultLevels();
  isoValues.setColorBarPosition("right");
```
I think that this may help to get proper levels.

Reply:
All `Contour` class constructors that do not have levels as an argument construct the default levels and labels by calling `buildDefaultLevels` and `buildDefaultLabels`. Recalling these methods after constructing the contour would be duplicative.
On the other hand, the `plot_graph_loglikelihood_contour.py` example is no longer adapted to the new contour functionalities. I will edit it to make it more relevant.

Reply:
You are right. I did not see that:

https://github.com/openturns/openturns/blob/8d44bc24665215d3e15f50d0752e83583e94c969/lib/src/Base/Graph/Contour.cxx#L139

So why are the default levels wrongly computed then?

Reply:
The levels are still correctly calculated as shown in Figures 2 and 3 where only the colors have changed. The poor distribution of colors is linked to the fact that the function takes very negative values occasionally and is close to -13 over the majority of the surface represented.
Figures 4 and 5 are constructed from Figure 3. The error comes from the fact that Figure 3 is no longer made up of N contours each having a single level, but of a single contour with N levels. This error is corrected in the new version of the example.